### PR TITLE
Show message about loading module in status bar

### PR DESCRIPTION
**Pull request by dimitry-**
Show the status bar and show progress messages while loading it.

Bug: http://b/162705265
Test: clear cache, start Orbit, load symbols for multiple modules
      check that messages in status bar showed and updated according
      on what Orbit is currently doing.

### CODE REVIEW

File: OrbitQt/orbitmainwindow.cpp [hunk 1]
```diff
@@ -64,6 +66,12 @@ OrbitMainWindow::OrbitMainWindow(QApplication* a_App, ApplicationOptions&& optio
   ui->HomeHorizontalSplitter->setSizes(sizes);
   ui->splitter_2->setSizes(sizes);
 
+  QStatusBar* status_bar = new QStatusBar(this);
```
Comment (reichlfl) [inline OrbitQt/orbitmainwindow.cpp:69]: I'd prefer to define this in the ui file and not create it in code.

File: OrbitQt/orbitmainwindow.h [hunk 2]
```diff
@@ -98,7 +99,8 @@ class OrbitMainWindow : public QMainWindow {
   QIcon icon_start_capture_;
   QIcon icon_stop_capture_;
 
-  std::string m_CurrentPdbName;
```
Comment (reichlfl) [inline OrbitQt/orbitmainwindow.h:101]: Was this intentionally removed, seems quite unrelated to the PR?
Apparently it was unneeded, so it's fine for me, just want to make sure.

File: OrbitQt/orbitmainwindow.cpp (hunk 1 above)
Comment (dimitry-) [reply OrbitQt/orbitmainwindow.cpp:69]: I think this is supposed to be created in the code? At least I did not see anything it designer. If it needs to be done in UI which section should this be added to?

File: OrbitQt/orbitmainwindow.h (hunk 2 above)
Comment (dimitry-) [reply OrbitQt/orbitmainwindow.h:101]: Not related no - but since it was unused I removed it.

File: OrbitQt/orbitmainwindow.cpp (hunk 1 above)
Comment (reichlfl) [reply OrbitQt/orbitmainwindow.cpp:69]: It's a bit hidden. In the designer: Right-click the main window, select "Create status bar". Then you can access the status bar in code through `OrbitMainWindow::statusBar()`. The Designer then adds a `<widget class="QStatusBar" name="statusBar"/>` directly as child of the QMainWindow.

Comment (dimitry-) [reply OrbitQt/orbitmainwindow.cpp]: dione.

File: OrbitQt/orbitmainwindow.ui [hunk 3]
```diff
@@ -223,7 +223,7 @@
      <x>0</x>
      <y>0</y>
      <width>976</width>
-     <height>22</height>
```
Comment (dimitry-) [inline OrbitQt/orbitmainwindow.ui:226]: hmm. I'm not sure about this one.

Comment (reichlfl) [reply OrbitQt/orbitmainwindow.ui:226]: @beckerhe I think this should be the same as what happened to you when you cleaned up the UI files, right?

**Comment by dimitry-** [comment]
ops. I forgot to rebase - I guess we are stuck with this patchset commit now :(
